source/features/unfinished-comments.tsx

```diff
@@ -0,0 +1,30 @@
+import select from 'select-dom';
+import * as pageDetect from 'github-url-detection';
+
+import features from '.';
+
+let documentTitle = '';
+
+async function updateDocumentTitle(): Promise<void> {
+	if (document.visibilityState === 'hidden') {
+		if (select.all('textarea').some(textarea => textarea.value.length > 0 && (textarea.offsetWidth > 0 || textarea.offsetHeight > 0))) {
+			console.log('changing title');
+			documentTitle = document.title;
+			document.title = '(Draft comment) ' + document.title;
+		}
+	} else {
+		document.title = documentTitle;
```

Comment:
We should only change the title if we changed it, and then we should reset it. The title might be changed by GitHub as well, so we don’t want to cache it for long.

Reply:
This should be fixed now, I think.